ci/run-sanity.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+cd "$(dirname "$0")/.."
+
+rm -f config/run/init-completed
+
+timeout 15 ./run.sh &
+pid=$!
+
+attempts=10
+while [[ ! -f config/run/init-completed ]]; do
```

Comment:
By the way, It's a shame we couldn't do these kinds of common tasks relatively easily in shell scripts in this age.

Basically, there is no simple construct for timeouts/retries. However, out of my serendipity, I came up with this:

```bash
curl --retry 60 --retry-delay 1 file://$(realpath config/run/init-completed)
```
But it didn't work...

Although it's odd usecase for curl, but might be sexy to able to do so, I'll probably submit a PR to the mighty swiff-knife `curl`.

Reply:
ha yeah that would be super cool!

Reply:
FYI: As promised, I've just created a [PR for the `curl`](https://github.com/curl/curl/pull/4656)! Let's see what this turns out. :)

Reply:
Congrats! The upstream `curl` finally added `--retry-all-errors`. Still ubuntu haven't updated it even in the `groovy` release channel, though. We just have to wait more 1.5 years for the next LTS (at least?).

https://curl.haxx.se/changes.html#7_71_0
https://daniel.haxx.se/blog/2020/06/24/curl-7-71-0-blobs-and-retries/
https://curl.haxx.se/docs/manpage.html#--retry-all-errors